sklearn/feature_extraction/tests/test_feature_hasher.py

```diff
@@ -106,3 +106,44 @@ def test_hasher_zeros():
     # Assert that no zeros are materialized in the output.
     X = FeatureHasher().transform([{'foo': 0}])
     assert_equal(X.data.shape, (0,))
+
+
+def test_hasher_alternate_sign():
+    X = [["foo", "bar", "baz", "investigation need", "records"]]
+    # the last two tokens produce a hash collision that sums as 0
```

Comment:
Can you move this comment above the previous line?

(+1 for a well thought of test case. Thx)